### PR TITLE
Message Of  The Pull Request

### PR DESCRIPTION
**Pull request by mariusbaciu**
This is the feature test to check that the pull request are okish

### CODE REVIEW

File: 1.txt [hunk 1]
```diff
@@ -8,4 +8,7 @@
 7
 1000
 line 1
-line 2
\ No newline at end of file
+line 2
+pull request line 1
+pull request line 2
```
Comment (MihaiMariusBaciu) [inline 1.txt]: Please Change This with a better logic

File: 1.txt [hunk 2]
```diff
@@ -8,4 +8,7 @@
 7
 1000
 line 1
-line 2
\ No newline at end of file
+line 2
+pull request line 1
```
Comment (MihaiMariusBaciu) [inline 1.txt]: This is yet another comment so we can check things up
logo2

File: 1.txt [hunk 3]
```diff
@@ -8,4 +8,7 @@
 7
```
Comment (MihaiMariusBaciu) [inline 1.txt]: This is a mess please change this as well

**Comment by MihaiMariusBaciu** [comment]
I do not like what you are saying about my code

**Comment by mariusbaciu** [comment]
I think We should change this with another feature

**Comment by mariusbaciu** [comment]
Or Maybe Use Another Thing That is Yet Another Stuff

**Comment by mariusbaciu** [comment]
Why are you not using this since you should

**Comment by MihaiMariusBaciu** [comment]
Don't press Random Buttons

**Comment by mariusbaciu** [comment]
I don't like you

**Pull request closed without merging**